### ISSUE TITLE
Add CI/CD support [Travis]

### ISSUE THREAD

**Issue by mdolata** [mdolata/TicTacToe#1]
I need help with configure simple pipelines using Travis.
I'd like to have automatic test run after every push, every push to master and on every PR

**Comment by alistairholmes** [comment]
Hi there :wave: I would like to be assigned to this please :smile:

**Comment by alistairholmes** [comment]
Do you already have a Travis account?

**Comment by mdolata** [comment]
Yes, I've added this project to `travis-ci.com` in settings page. Let me know if you need more

**Comment by mdolata** [comment]
@alistairholmes I cant accept your PR. 

1. This project is not related to android. It is just pure Java code
2. The build failed... For me the main purpose of CI is automatic checks for each ie PR 

Please let me know if you still want to work on it

**Comment by alistairholmes** [comment]
@mdolata  I'm so sorry about that. It looked just like an android project. I'll work on a fix for it. And yes I understand, the CI is working for checking PRs as it did that for my PR.

**Comment by fadeltd** [comment]
Why not use github actions instead?

**Comment by Kineolyan** [comment]
The main issue was that it was required to generate the wrapper before calling `gradlew`.
My PR #10 fixes that.
Now, @fadeltd suggested you switched to Github actions, currently in beta. If you still want to pursue Travis, this PR should be ok.
I feel I could submit my PR even though @alistairholmes  started one because it's been a while without update and you may want your build eventually.

Cheers

**Comment by mdolata** [comment]
@Kineolyan feel free to submit your PR with Travis.

**Comment by Kineolyan** [comment]
@mdolata I already did 😄
